src/Microsoft.Health.Fhir.Core/Features/Security/ClaimsPrincipalExtractor.cs

```diff
@@ -10,14 +10,14 @@
 using Microsoft.Health.Fhir.Core.Configs;
 using Microsoft.Health.Fhir.Core.Features.Context;
 
-namespace Microsoft.Health.Fhir.Core.Features.Persistence
+namespace Microsoft.Health.Fhir.Core.Features.Security
 {
-    public class ClaimsIndexer : IClaimsIndexer
+    public class ClaimsPrincipalExtractor : IClaimsExtractor
```

Comment:
Maybe `PrincipalClaimsExtractor`?

Reply:
That's what I had originally but I renamed it because the .NET type is called ClaimsPrincipal but I guess we are extracting the Claims so PrincipalClaimsExtractor makes more sense.